ci/nightly/pipeline.yml

```diff
@@ -77,7 +77,7 @@ steps:
           - { value: parallel-workload-dml }
           - { value: parallel-workload }
           - { value: parallel-workload-100-threads }
-          #- { value: parallel-workload-cancel }
```

Comment:
Leftover?

Reply:
Yes, forgot this in a previous change, just noticed.